shell/executables.ts

```diff
@@ -0,0 +1,27 @@
+import { $ } from './run.ts';
+
+/**
+ * Makes a file executable in Linux/Unix/MacOS environments.
+ *
+ * This includes setting the executable bit on the file, and if on MacOS, unsetting the Apple quarantine bit applied to
+ * downloaded files.
+ *
+ * Note: This method doesn't do anything for Windows. This is because Windows typically decides whether a file is
+ * executable based on its file extension.
+ *
+ * @param executablePath The path to the file to make executable
+ */
+export async function makeFileExecutable(executablePath: string): Promise<void> {
```

Comment:
I think it would be useful to have the quarantine removal be placed in its own special function rather than doing it as part of the general `makeFileExecutable` flow. When using this function for its "headline" purpose  I might not expect that it would also quietly remove Apple's quarantine protection.

I would also add a comment to the new de-quarantine function indicating that you should only use it advisedly. It's sort of an unsafe function in general usage.

Reply:
Done.